ArduCopter/mode_turtle.cpp

```diff
@@ -0,0 +1,138 @@
+#include "Copter.h"
+
+#if MODE_TURTLE_ENABLED == ENABLED
+
+#define CRASH_FLIP_EXPO 35.0f
+#define CRASH_FLIP_STICK_MINF 0.15f
+#define power3(x) ((x)*(x)*(x))
+
+// throw_init - initialise turtle mode controller
```

Comment:
is this a copy-paste mistake? What do you mean by throw_init?

Reply:
Yes copy and paste!

Reply:
Fixed